### PR TITLE
Bry/doc delete firebase

### PR DESCRIPTION
**Pull request by bry-gavino**
# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Working on ability to delete uploaded docs from firebase.

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] have all firebase docs appear in upload screen
 - [x] update firebase accordingly when hitting "Upload" (Images that are in the imageUris array ONLY should be in the firebase when updated. Thus, hitting the "X" button on an image should delete it from the imageUris array and when "Upload" is hit, it should be removed from the firebase)

## How can the reviewer test your code? 👩‍💻
_example: run `expo start` and make sure that image docs from the firebase appear to their respective docs & cases

Screen Shot 2022-05-17 at 4 21 22 PM
Screen Shot 2022-05-17 at 4 21 36 PM

🧜‍♀️ cc: @shannonbonet @phoebeli23

### CODE REVIEW

**Review by shannonbonet — COMMENTED** [review]
Couple of bugs: 

Adding docs: 
- cannot select pics to upload 
- after taking 1 photo, cannot add more photos 

Deleting docs: 
- works great, but cannot add any now

Style changes: 
- make upload and delete icons clickable (low priority)

File: src/components/ImagePicker/ImagePicker.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const ImagePicker = ({ navigation }: any) => {
 
   const widgetSettings = useMemo(
     () => ({
-      getImageMetaData: false, // true might perform slower results but gives meta data and absolute path for ios users
+      getImageMetaData: true, // true might perform slower results but gives meta data and absolute path for ios users
```
Comment (shannonbonet) [inline src/components/ImagePicker/ImagePicker.tsx:36]: How do your changes make use of the metadata?

Comment (bry-gavino) [reply src/components/ImagePicker/ImagePicker.tsx:36]: Ty for pointing this out -- I forgot to change the value back to false.

I tested this locally and the uploading feature works just fine. I'll edit this file right now.

**Comment by bry-gavino** [comment]
package-lock.json has upgraded versions and node and expo. We can downgrade back to the main branch's versions when merging.

File: src/components/ImagePicker/ImagePicker.tsx (hunk 1 above)
Comment (shannonbonet) [reply src/components/ImagePicker/ImagePicker.tsx]: maybe fix in a separate packages PR? 
https://stackoverflow.com/questions/55311228/how-to-remove-warning-async-storage-has-been-extracted-from-react-native-core

**Pull request merged**